gemfiles/rails_5_2.gemfile.lock

```diff
@@ -51,6 +51,7 @@ GEM
       thor (>= 0.14.6)
     concurrent-ruby (1.1.7)
     crass (1.0.6)
+    docile (1.4.0)
```

Comment:
Where did this one came from?

Reply:
Seems like it's one of `simplecov`'s dependencies.
https://github.com/simplecov-ruby/simplecov/blob/efdb08db63b35577b3b0d79db893ead1d848d8dd/simplecov.gemspec#L37